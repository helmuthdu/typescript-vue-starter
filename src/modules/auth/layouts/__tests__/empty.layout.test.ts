import { createLocalVue, shallowMount, Wrapper } from '@vue/test-utils';
import VueRouter from 'vue-router';
import EmptyLayout from '../empty.layout.vue';

const localVue = createLocalVue();
const router = new VueRouter();

localVue.use(VueRouter);

describe('About route component', () => {
  let wrapper: Wrapper<EmptyLayout>;

  beforeEach(() => {
    wrapper = shallowMount(EmptyLayout, { localVue, router });
  });

  it('should renders with props', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
