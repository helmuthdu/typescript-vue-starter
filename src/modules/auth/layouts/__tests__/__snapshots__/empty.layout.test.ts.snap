// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About route component should renders with props 1`] = `
<div>
  <router-view-stub name="default"></router-view-stub>
</div>
`;
